packages/math-input/src/components/input/scroll-into-view.ts

```diff
@@ -58,15 +46,17 @@ export const scrollIntoView = (containerNode, keypadNode) => {
                 containerTopPx,
             );
 
-            scrollNode.scrollTop += scrollOffset;
+            if (scrollNode) {
```

Comment:
Logic change: `document.scrollingElement` can return null, so I had to protect against that for TypeScript.